dstk-infra/apollo/src/graphql/model-version/modelVersion.ts

```diff
@@ -3,35 +3,37 @@ import { MLModel, ObjectionMLModel } from '../model/model.js';
 import { Model } from 'objection';
 import { ObjectionStorageProvider } from '../storage-provider/storageProvider.js';
 
-export const MLModelVersion = builder.objectRef<ObjectionMLModelVersion>('MLModelVersion').implement({
-    fields: (t) => ({
-        modelVersionId: t.field({
-            type: 'ID',
-            resolve(root: ObjectionMLModelVersion, _args, _ctx) {
-                return root.$modelClass.idColumn[0];
-            },
-        }),
+export const MLModelVersion = builder
```

Comment:
IDK why husky decided to pick up these lints now but 🤷

Reply:
What was the eslint complaining about?

Reply:
I didn't even notice it, the recommit hook just auto formatted this when I made the commit